### ISSUE TITLE
error transactions not reported as failed in kibana

### ISSUE THREAD

**Issue by brettmc** [elastic/apm-agent-php#463]
**Describe the bug**
When a traced transaction fails, I can see errors reported in the kibana transaction, and the errors are listed in various places in kibana. However, the failure rate ("error rate") for the related named transaction is reported as 'N/A'.
![Screenshot from 2021-08-03 10-00-01](https://user-images.githubusercontent.com/4978962/127937824-23dbba09-202b-4dc9-acf2-a770a79c8661.png)

The transaction result is "HTTP 5xx" (set by the agent, and matches what [the documentation](https://github.com/elastic/apm-server/blob/7.0/docs/spec/transactions/transaction.json#L52) suggests it should be):
```json
{
  "transaction": {
    "name": "GET /v1/units/current",
    "type": "request",
    "id": "2fc8fab5c983f7eb",
    "trace_id": "eb3823b12ecb575008c395fef73f455a",
    "timestamp": 1627947687069486,
    "duration": 180.013,
    "span_count": {
      "started": 4
    },
    "result": "HTTP 5xx",
    "context": {
      "tags": {
        "trackingId": "5132276c-6889-45da-b2c0-7c3ea21d8595",
        "sessionId": "",
        "username": ""
      },
      "request": {
        "method": "GET",
        "url": {
          "hostname": "127.0.0.1:10403",
          "full": "http://127.0.0.1:10403/v1/units/current",
          "raw": "http://127.0.0.1:10403/v1/units/current",
          "pathname": "/v1/units/current",
          "port": 10403,
          "protocol": "http",
          "search": ""
        }
      }
    }
  }
}
```
![Screenshot from 2021-08-03 09-55-00](https://user-images.githubusercontent.com/4978962/127937566-4a7fab50-c82b-4889-be88-3743b3491ff5.png)

I found this comment: https://github.com/elastic/kibana/issues/77864#issuecomment-861479413 which indicates that the error rate is derived from "event.outcome=failed" - but when I located the event in kibana discover, I see that "event.outcome=unknown" - we are using elastic v7.13.1
![Screenshot from 2021-08-03 09-52-54](https://user-images.githubusercontent.com/4978962/127937375-1ed3dc42-534c-4dc2-8eba-9a417490f23e.png)

What should be setting "event.outcome" ? The agent? Is there anything I can do in userland code to set this myself?

**Expected behavior**
I am able to mark transactions as failed, and observe failure rates etc in kibana APM

**Comment by brettmc** [comment]
@michaelhyatt as discussed

**Comment by brettmc** [comment]
fixed by https://github.com/elastic/apm-agent-php/pull/441 in v1.2 which was recently released - I was still using agent v1.1
